data/models/FAH_SARS-CoV-2_spike.yml

```diff
@@ -0,0 +1,10 @@
+name: FAH_SARS-CoV-2_spike
+description: PDB of the SPIKE protein in the closed state
+Url: s3://fah-public-data-covid-19-cryptic-pockets/SARS-CoV-2/spike/prot_masses.pdb
```

Comment:
This URL corresponding to this S3 path should be
```
url: http://fah-public-data-covid-19-cryptic-pockets.s3-website-us-east-1.amazonaws.com/SARS-CoV-2/spike/prot_masses.pdb
```
Can you check I have the right region (`us-east-1`) for the bucket?

Reply:
Ahh, thats good! I checked and think we're in us-east-2.

`https://fah-public-data-covid19-cryptic-pockets.s3.us-east-2.amazonaws.com/SARS-CoV-2/spike/prot_masses.pdb`